src/runtime/recipe/handle-connection.ts

```diff
@@ -17,23 +17,23 @@ import {compareArrays, compareComparables, compareStrings} from './comparable.js
 export type Direction = 'in' | 'out' | 'inout' | 'host';
 
 export class HandleConnection {
-  _recipe: Recipe;
-  _name: string;
-  _tags: string[] = [];
+  private readonly _recipe: Recipe;
+  private readonly _name: string;
+  private _tags: string[] = [];
   private resolvedType: Type | undefined = undefined;
-  _direction: Direction | undefined = undefined;
-  _particle: Particle;
+  private _direction: Direction | undefined = undefined;
+  private _particle: Particle;
   _handle: Handle | undefined = undefined;
```

Comment:
Question: What is the default level of access in typescript?

Reply:
Default is public.  There are tslint rules that require a visibility, but we have not adopted them.